apps/emqx_management/src/emqx_mgmt_api_listeners.erl

```diff
@@ -158,21 +192,12 @@ fields(listener_id) ->
         {id,
             ?HOCON(atom(), #{
                 desc => "Listener id",
-                example => 'tcp:demo',
```

Comment:
Maybe don't change this value. to make sure user add  or delete a noexist listeners.